src/test/java/Tests/UpdateProfileTests.java

```diff
@@ -0,0 +1,212 @@
+	LoginPage loginPage ;
+	HomePage homePage;
+	UserProfilePage userProfilePage;
+	
+	@BeforeClass
+	public void driversetup() {
+		loginPage = new LoginPage(driver);
+		homePage = new HomePage(driver);
+        userProfilePage = new UserProfilePage(driver);
+        
+        }
+	
+	@Test(priority=0, description = "Verifying Updating Profile Pic  Cancel Button")
+	public void VerifyUpdatingProfilePicCancelButton() throws Exception{
+		loginPage.login("addy3@mailinator.com", "password");
+		Thread.sleep(5000);
+		homePage.Dropdown.click();
+		userProfilePage.btnEditProfile.click();
+		userProfilePage.btnEditprofilepic.click();
+		userProfilePage.btnUploadfile.sendKeys("C:\\\\Users\\\\admin\\\\Desktop\\\\mypic.jpg");
+		//userProfilePage.btnUploadfile.sendKeys("D:\\Web Automation\\AutomationTests_EduthrillWebapp\\src\\test\\resources\\data");
+		userProfilePage.btnCancel.click();
+		
+		}
+	
+	@Test(priority=1, description = "Verifying Updating Profile Pic")
+	
+		public void VerifyUpdatingProfilePic()throws Exception{
+		userProfilePage.btnEditprofilepic.click();
+		userProfilePage.btnUploadfile.sendKeys("C:\\Users\\admin\\Desktop\\mypic.jpg");
+		Thread.sleep(1000);
+		userProfilePage.btnUploadPic.click();
+		Thread.sleep(2000);
+		
+	}
+	
+	
+	
+	@Test(priority=2, description = "Verifying Success Rate Is Getting Displayed")
+    public void VerifySuccessRateDisplaying()throws Exception{
+		Thread.sleep(1000);
+		Assert.assertTrue(userProfilePage.fetchSuccessRate());
+	}
+	
+	@Test(priority=3, description = "Verify Number Of Games Played Is Displaying ")
+	public void VerifyNumberOfGamesPlayedDisplaying()throws Exception{
+		Assert.assertTrue(userProfilePage.fetchCountOfGamesPlayed());
+		
+		}
+	@Test(priority=4, description = "Verify Member Since Count Is Displaying ")
+	public void VerifyMemberSinceFieldIsDisplaying()throws Exception{
+		Assert.assertTrue(userProfilePage.fetchMemberSince());
+		
+	}
+	
+	@Test(priority=5, description = "Verify Pack Details Are Displaying")
+	public void VerifyPackDetailsAreDisplaying()throws Exception{
+		Assert.assertTrue(userProfilePage.fetchUserPacksCount());
+		
+	}
+	
+	@Test(priority=6, description = "Verify Badges are getting displayed")
+	public void VerifyBadgesAreGettingDisplayed()throws Exception{
+		Assert.assertTrue(userProfilePage.userBadges.isDisplayed());	
+		
+	}
+	
+	@Test(priority=7, description = "Verifying Updating Company Details")
+	public void VerifyUpdatingCompanyDetails() throws Exception{
+		userProfilePage.editCompany.click();
+		userProfilePage.clearCompanyFields();
+		userProfilePage.fieldComapany.sendKeys("MyCompany");
+		userProfilePage.fieldJobDesignation.sendKeys("CEO");
+		userProfilePage.saveCompanyDetailsButton.click();
+		Thread.sleep(5000);
+	}
+	
+	
+	
+	
+	@Test(priority=8, description = "Verifying Updating College with Missing College Name")
+	public void VerifyUpdatingCollegeDetailsWithMissingCollegeName() throws Exception{
+		userProfilePage.btnEditEducation.click();
```

Comment:
Create one common method for updatingcollege details and use it here and all other college related tcs.